contracts/src/BackedCommunityTokenDescriptorV1.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.12;
+
+import {IBackedCommunityTokenDescriptorV1} from "./interfaces/IBackedCommunityTokenDescriptorV1.sol";
+
+contract BackedCommunityTokenDescriptorV1 is IBackedCommunityTokenDescriptorV1 {
+    uint256 constant activityCategoryId = 0;
```

Comment:
should sync with @jringenberg on these as I think he might be cutting some of this 😬

Reply:
@adamgobes I added some thinking on raw-counters to the scope doc as a decision. I've gone back and forth and am open to more reasons (one of which could very legitimately be tech investment already done!), so please let me know what you think. [Link to Notion comment](https://www.notion.so/Community-NFT-497549140b5d4fcb820f613def3b7f69?d=3b8ea87554a1429c8d96170b0d08a47f#3d958b6ea1d942b6add2cc207cfe5775)

Reply:
@jringenberg no big tech investments would be lost in removing the some of the counters. should be very easy to adhere to the updated scope

@wilsoncusack my idea with this descriptor is that it is very specific to the current scope. Storage for big XP things and things like pull requests right now are pretty similar, just a counter. The only thing thats different is the way they are rendered.